### PR TITLE
Migrated Nightly pipeline to 1ES Hosted Pools

### PR DESCRIPTION
**Pull request by gapra-msft**
TODO : Debian 9

### CODE REVIEW

File: blobfuse2-nightly.yaml [hunk 1]
```diff
@@ -48,7 +42,7 @@ parameters:
     default: false
 
 jobs:
-  # Ubuntu based test suite
+  # Ubuntu Tests
   - job: Set_1
```
Comment (vibhansa-msft) [inline blobfuse2-nightly.yaml:46]: We need to add Ubn-22 here.

Comment (vibhansa-msft) [reply blobfuse2-nightly.yaml:46]: For Debb-9 we can add one more set and have our self-hosted agent for now? or we want to move completely to 1ES and see it later?

Comment (gapra-msft) [reply blobfuse2-nightly.yaml:46]: yes I can add that

Comment (gapra-msft) [reply blobfuse2-nightly.yaml:46]: Done

**Review by vibhansa-msft — APPROVED** [review]

**Review by souravgupta-msft — APPROVED** [review]

**Review by tasherif-msft — APPROVED** [review]

**Pull request merged**